include/itkImageToTorchImageFilter.hxx

```diff
@@ -71,11 +72,18 @@ ImageToTorchImageFilter< TInputImage, TOutputTorchImage >
   itkAssertOrThrowMacro( inputSize == outputSize, "ImageToTorchImageFilter: input and output images must have the same size in each dimension");
 
   // Do the memory copy of the whole image.
-  // Do we need to Allocate() the output?!!!
-
-  torch::Tensor inputTensor =
-    torch::from_blob( ( void * )inputBufferPointer, outputTorchImage->m_Tensor.sizes(), outputTorchImage->m_Tensor.options().device( torch::kCPU ) );
-  outputTorchImage->m_Tensor = inputTensor.to( outputTorchImage->m_Tensor.options() );
+  const std::vector< int64_t > torchSize = outputTorchImage->ComputeTorchSize();
+  const c10::TensorOptions commonTensorOptions = torch::dtype( Self::OutputTorchImageType::TorchValueType ).layout( torch::kStrided );
```

Comment:
I assume that `torch::kStrided` here means contiguous in memory?

Reply:
Yes.